### ISSUE TITLE
UI input to display label of discrete covariate at average position of samples

### ISSUE THREAD

**Issue by kevinrue** [iSEE/iSEE#422]
**Describe the solution you'd like**
A ui input to select a discrete covariate to display at the average position of samples in reduced dimensions.

**Describe alternatives you've considered**
A checkbox to show/hide labels.

**Additional context**

**Comment by LTLA** [comment]
Yes, I realized we also need this for the clusters. Too many damn colors.

**Comment by kevinrue** [comment]
Sorry that I left that hanging, but i was planning to do this, rather: https://rdrr.io/cran/ggforce/man/geom_mark_ellipse.html

**Comment by LTLA** [comment]
Yeah, not sure I would have liked the extra dependency, plus there is quite a bit of visual overhead.
